README.md

```diff
@@ -176,3 +176,12 @@ Issues for first timers: https://github.com/TechSquidTV/Shoutify/contribute
 Built on: Go
 
 https://github.com/KyleTryon/Feed-Monitor
+
+## LinuxGamer/Universe
+
+> An open source app to do anything and everything.
```

Comment:
I see what you're trying to do, but I think this description is too broad.

Could you phrase it in a more specific way?

Reply:
How does it look now?

Reply:
looks good! thank you